src/utils/keystore.ts

```diff
@@ -141,8 +188,8 @@ function marshal (name, derivedKey, privateKey, nonce, salt, options = {}) {
   )
 }
 
-export function getAddressFromPriv (secret) {
-  const keys = nacl.sign.keyPair.fromSecretKey(str2buf(secret))
+export function getAddressFromPriv (secret: string|Buffer): string {
+  const keys = nacl.sign.keyPair.fromSecretKey(typeof secret === 'string' ? str2buf(secret) : secret)
```

Comment:
Since this is also exported from `src/utils/crypto.js`, let's remove it and import the other one which includes docs